kafka_topic-transferer/kafka-topic-transferer.go

```diff
@@ -0,0 +1,85 @@
+package transferer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	kf "github.com/codeuniversity/smag-mvp/kafka"
+	"github.com/codeuniversity/smag-mvp/models"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+
+	"github.com/segmentio/kafka-go"
+)
+
+// Transferer represents the Transferer containing all clients it uses
+type Transferer struct {
+	fromTopic *kafka.Reader
+	toTopic   *kafka.Writer
+
+	*service.Executor
+}
+
+// New returns an initilized Transferer
+func New(fromTopic string, toTopic string) *Transferer {
+	t := &Transferer{}
+
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.MustGetStringFromEnv("KAFKA_GROUPID")
+	readerConfig := kf.NewReaderConfig(kafkaAddress, groupID, fromTopic)
+	writerConfig := kf.NewWriterConfig(kafkaAddress, toTopic, true)
+
+	t.fromTopic = kf.NewReader(readerConfig)
+	t.toTopic = kf.NewWriter(writerConfig)
+
+	t.Executor = service.New()
+
+	return t
+}
+
+//Run the Transferer
+func (c *Transferer) Run() {
+	fmt.Println("Start Transferer")
```

Comment:
you are missing the `defer c.MarkAsStopped()` here (or something like that)
-> if we get #33 merged, we can simplify this greatly here :)